ext/odbc/php_odbc.c

```diff
@@ -389,19 +452,39 @@ PHP_MINIT_FUNCTION(odbc)
 #endif
 
 	REGISTER_INI_ENTRIES();
-	le_result = zend_register_list_destructors_ex(_free_odbc_result, NULL, "odbc result", module_number);
-	le_conn = zend_register_list_destructors_ex(_close_odbc_conn, NULL, "odbc link", module_number);
 	le_pconn = zend_register_list_destructors_ex(NULL, _close_odbc_pconn, "odbc link persistent", module_number);
```

Comment:
Note, if we are keeping this the
>  Persistent resources that are only used internally and not exposed to userland:

list of https://github.com/php/php-tasks/issues/6 needs updating